docs/management.md

```diff
@@ -418,6 +446,66 @@ management.oauth_provider_url = https://my-web-portal
 With the previous settings, the management UI exposes the HTTP endpoint `/login` which accepts `content-type: application/x-www-form-urlencoded` and it expects the JWT token in the `access_token` form field. This is the endpoint where the Web portal will redirect users to the management UI.
 Additionally, RabbitMQ also accepts a JWT token in the HTTP `Authorization` header when the user lands on the management UI.
 
+With `sp_initiated` logon types, there is no need to configure the `oauth_provider_url` if `auth_oauth2.issuer` was set. However, for `idp_initiated` flows the `auth_oauth2.issuer` url may not necessarily be the url where to send users to authenticate. When this occurs, the `management.oauth_provider_url` overrides the `auth_oauth2.issuer` url.
+
+### Support multiple OAuth 2.0 resources {#support-multiple-resources}
+
+RabbitMQ 3.13 introduced support for multiple OAuth 2.0 resources in the OAuth 2.0 plugin and in the Management plugin.
+
+Once you have configured the [OAuth 2.0 plugin](./oauth2#multiple-resource-servers) with all the required OAuth 2.0 resources, they are automatically supported by the management UI.
+
+Given the following OAuth 2.0 plugin configuration which consists of four OAuth 2.0 resources:
+```ini
+auth_oauth2.issuer = http://some_idp_url
+auth_oauth2.scope_prefix = rabbitmq.
+auth_oauth2.resource_servers.1.id = rabbit_prod
+auth_oauth2.resource_servers.2.id = rabbit_dev
+auth_oauth2.resource_servers.3.id = rabbit_qa
+auth_oauth2.resource_servers.4.id = rabbit_internal
+```
+
```

Comment:
Is the above an example, if so maybe put it under an "Example" heading then above rewrite to: Let's take the following OAuth 2.0 plugin configuration which consists of four OAuth 2.0 resources as an example.

Reply:
but this configuration is preceded with the sentence `Let's say you have the following OAuth 2.0 plugin configuration which consists of four OAuth 2.0 resources:`. it think it is clear that this is an example configuration.  

Yes, all configuration is done on the same location `rabbitmq.conf'`. All the configuration done in this section or in other sections unrelated to oauth2 are stored in the same file.